interfaces/generate_test.go

```diff
@@ -9,6 +9,8 @@ import (
 	"testing"
 
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/appconfiguration/armappconfiguration/v2"
+	resourcesarmmanagedapplications "github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armmanagedapplications"
```

Comment:
Before this PR, we were taking the last part of the import part, which means `resources -> armmanagedapplications` and `solutions -> armmanagedapplications` overlapped